### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
removed Duplicates

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6595

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -94,13 +94,6 @@ For most subscriptions modern authentication is automatically turned on, but if
 2. Under the **Services** tab, choose **Modern authentication**, and in the **Modern authentication** pane, make sure **Enable Modern authentication** is selected. Choose **Save changes**.
 
 
-### Turn off legacy per-user MFA
-
-If you have previously turned on per-user MFA, you must turn it off before enabling Security defaults.
-
-1. In the Microsoft 365 admin center, in the left nav choose **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">**Active users**</a>.
-1. On the **Active users** page, choose **Multi-factor authentication**.
-1. On the multi-factor authentication page, select each user and set their Multi-Factor auth status to **Disabled**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**